<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Pardusman-wizard</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>450</y>
      <width>695</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title" >
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFinish" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>buttonCancel</zorder>
    <zorder>buttonBack</zorder>
    <zorder>buttonNext</zorder>
    <zorder>buttonFinish</zorder>
    <zorder>horizontalSpacer</zorder>
   </widget>
   <widget class="QStackedWidget" name="pageStack" >
    <property name="geometry" >
     <rect>
      <x>161</x>
      <y>50</y>
      <width>524</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>524</width>
      <height>400</height>
     </size>
    </property>
    <widget class="QWidget" name="page" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>524</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="page_2" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pageDesc" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>20</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixSteps" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>80</y>
      <width>150</width>
      <height>373</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize" >
     <size>
      <width>150</width>
      <height>373</height>
     </size>
    </property>
    <property name="maximumSize" >
     <size>
      <width>32767</width>
      <height>373</height>
     </size>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
    <property name="indent" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="pageIcon" >
    <property name="geometry" >
     <rect>
      <x>28</x>
      <y>20</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize" >
     <size>
      <width>90</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize" >
     <size>
      <width>90</width>
      <height>50</height>
     </size>
    </property>
    <property name="text" >
     <string>PageIcon</string>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
